handbook/ce/index.md

```diff
@@ -1,6 +1,16 @@
 # Customer Engineering
 
-The Customer Engineering team partners with the [Sales team] (../sales/index.md) from a technical and product expertise perspective to ensure that all of Sourcegraph's prospects, customers, and users get the most possible value from it. This includes showing product demos, learning and understanding prospects' internal development environment and processes, helping companies integrate Sourcegraph into their engineering stack and internal processes to maximize value to their engineering teams, helping prospects run trials to recognize Sourcegraph value firsthand, building relationships with our accounts' champions and users, communicating the value of Sourcegraph, growing engagement and owning product training and technical support, and giving as many Sourcegraph "Aha!" moments as possible.
+On the Customer Engineering team, we help developers use Sourcegraph to solve their organization's Big Code problems. 
+
+- [**Customer Engineers**](roles/customer_engineer.md) help our developer users and champions get more value from using Sourcegraph.
```

Comment:
I don't feel super strongly about this but I did like the idea that the previous description highlighted about us really helping the customers realize the value. We help them use it but we also help them recognize how they could use it, if that makes sense?

Reply:
To "help our developer users and champions get more value from using Sourcegraph", I'll try adding "...in more and better ways". Does that capture it?

Reply:
I think so!